ext/hal/st/lib/sensor/vl53l1x/api/core/inc/vl53l1_api_calibration.h

```diff
@@ -0,0 +1,236 @@
+/*
+* Copyright (c) 2017, STMicroelectronics - All Rights Reserved
+*
+* This file is part of VL53L1 Core and is dual licensed,
+* either 'STMicroelectronics
+* Proprietary license'
+* or 'BSD 3-clause "New" or "Revised" License' , at your option.
```

Comment:
Please review https://docs.zephyrproject.org/latest/contribute/index.html#contributing-non-apache-2-0-licensed-components and fill out the README template.

cc: @avisconti

Reply:
@avisconti  Thanks!
I have a question that those code under ext/hal/st/ comes from ST. Can I modify the copyright part as vl53l0x does?

Reply:
@overheat 
For the ext part yes. You may refer to the existing VL53L0X for reference.
For the driver/sensor/vl53l1x you may change/add ur name and 2019 in the copyright section.

I think you may need to change CODEOWNER to cover that code. You should be the owner for the driver, while I think that someone from ST should for what concern the ext part (maybe myself or @erwango)

Reply:
@avisconti Thank you! I will update it. I will put you the CODEOWNER for the ext part, is that alright?

Reply:
@overheat 
OK, fine.

Reply:
@overheat 
I'm discussing with @erwango. I think that he can be the owner for it.

Reply:
So, at the end I think that you won't need to change CODEOWNERS at all.